ClientApp/src/hooks/useUser.tsx

```diff
@@ -36,6 +40,8 @@ const reducer = (state: IState, action: IReducerAction) => {
       return { ...state, location: action.data ?? null };
     case ReducerActionType.setRole:
       return { ...state, role: action.data ?? "Unspecified" };
+    case ReducerActionType.setInstitution:
+      return { ...state, institution: action.data ?? "Unspecified" };
```

Comment:
Should this be "Unspecified" or null?

Reply:
User without set metadata will not be affected